src/main.rs

```diff
@@ -41,12 +49,12 @@ extern crate jsonrpc_core;
 use std::env;
 use std::sync::Arc;
 
-mod actions;
-mod build;
-mod cmd;
-mod config;
-mod lsp_data;
-mod server;
+pub mod actions;
+pub mod build;
+pub mod cmd;
+pub mod config;
+pub mod lsp_data;
+pub mod server;
```

Comment:
I do not think they should be `pub`. Can you use `pub(crate)` or similar? I guess you changed it such that rustc complains about missing docs as otherwise they would be private.

Reply:
As one of the commit messages explains, these aren't *really* `pub` since this is a `[[bin]]` crate, not a `[lib]` crate, and these modules are top level, so it doesn't make these modules any more visible than they were before, just helps the lint out.